lib/registry.js

```diff
@@ -223,6 +223,10 @@ Registry.prototype.configureModel = function(ModelCtor, config) {
       'must be an object', modelName);
   }
 
+  if (typeof config.methods === 'object' && config.methods !== null) {
+    settings.methods = config.methods;
```

Comment:
Sorry, the `MyCustomModel` method should have been called `hello`. I forgot to change the name when copy-pasting :(